ui/app/templates/permissions.html

```diff
@@ -1,14 +1,24 @@
-<div loader ng-show="loading"></div>
+<div class="tabbable tabs-below">
+  <ul class="nav nav-pills">
+    <li ng-class="{ active: isSet(1) }">
+      <a href ng-click="setTab(1)">Users</a>
```

Comment:
isSet and setTab actually make this a bit tougher to read, because it requires looking at the template and the controller. Because they are such trivial functions (each contains only a single statement)< this would be a bit more readable if you got rid of them and inlined it instead. Eg:
```
    <li ng-class="{ active: (tab === 1) }">
      <a href ng-click="tab = 1">Users</a>
    </li>
```

Reply:
This is way better. I will effect change.